c8run/cmd/packager/main.go

```diff
@@ -0,0 +1,100 @@
+package main
+
+import (
+	"fmt"
+	"os"
+	"os/exec"
+	"strings"
+
+	"github.com/camunda/camunda/c8run/internal/packages"
+	"github.com/joho/godotenv"
+)
+
+func usage(exitcode int) {
+	fmt.Printf("Usage: %s [command] [options]\nCommands:\n package\n clean\n", os.Args[0])
+	os.Exit(exitcode)
+}
+
+func getBaseCommand() (string, error) {
+	if len(os.Args) == 1 {
+		usage(1)
+	}
+
+	switch os.Args[1] {
+	case "package":
+		return "package", nil
+	case "clean":
+		return "clean", nil
+	case "-h", "--help":
+		usage(0)
+	default:
+		return "", fmt.Errorf("unsupported operation: %s", os.Args[1])
+	}
+
+	return "", nil
+}
+
+func GetJavaVersion(javaBinary string) (string, error) {
```

Comment:
Potential refactoring follow up to move these function in a package as well (to be reused in "main main" + "packager/main")